src/Extensions/Abstractions/ReplayableLogs/ILogReplayer.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Replays activity logs if activity collects them</summary>
+	public interface ILogReplayer
+	{
+		/// <summary>
+		/// Replays activity logs if activity supports it
+		/// </summary>
+		/// <param name="activity"></param>
```

Comment:
>activity [](start = 19, length = 8)

Missing info

Reply:
Also added but for some reasons not visible https://github.com/microsoft/Omex/blob/feature/create-omex-extensions/src/Extensions/Abstractions/ReplayableLogs/ILogReplayer.cs